### DISCUSSION TITLE
Shelly Dimmer 2 without Neutral with Tasmota

### DISCUSSION THREAD

**Discussion by Frackel72** [arendst/Tasmota#11437] [Q&A] [answered]
Hello,
would like to operate a Shelly Dimmer 2 without a neutral conductor with Tasmota. After setting up, the dimmer is in the WiFi. If you switch it on, it immediately starts to switch itself on and off several times. He then carries out a reset. And the setup starts all over again.

process

Shelly Dimmer 2 flashed via OTA "Tasmota 9.3.1"

Setting up the Tasmota software for Shelly Dimmer 2
" {"NAME":"Shelly Dimmer 2","GPIO":[0,3200,0,3232,5568,5600,0,0,193,0,192,0,320,4736],"FLAG":0,"BASE":18}"

Upgrade additional firmware "shelly-dimmer-stm32_v51.5.bin"

consumer
5x 3,5w Phillips LED dimmable GU 10

Does anything else have to be set in the software?
Please help.....me

At the moment the dimmer is running stable with the Shelly firmware.

Greetings
Frackel72

**Comment by Jason2866** [comment]
Do you have installed the bypass?

**Reply by Frackel72** [reply]
Bypass is not installed, the dimmer has a minimum load of 10W, 17.5W are currently connected

**Reply by Jason2866** [reply]
Try with bypass. 17.5W LED load is not much. Anyway Shelly Dimmer is not working well without neutral

**Reply by sfromis** [reply]
Unless the 17.5W is an old-fashioned resistive load, you're very likely to need a bypass.

**Comment by barbudor** [comment, marked as answer]
Shelly Dimmer with no neutral is a fail
Not related to Tasmota
Even with original firmware it doesn't work well.
This has been discuss on the repo of the alternate STM32 firmware. 

I've given up on this use case. 
Better install it at the bulb level with neutral and use control it from a remote (tasmotized) switch

**Reply by Frackel72** [reply]
when I connect the dimmers to the lamps I need 5 dimmers ...... not good 
This is my front door lighting. The cables all come from a solid concrete ceiling ...
The Shelly is installed in the hallway, and installing an extra neutral conductor here entails a not inconsiderable need for renovation ..... I want to avoid it. Because you can no longer get the wall coating made of latex plastic today.

**Reply by Jason2866** [reply]
So Shelly dimmer is not the way. It will not work with that small load without the bypass.
It **might** work with the bypass. In my tests it never worked nice without neutral.

**Reply by Frackel72** [reply]
Because I let it continue to run with the original firmware, they can do it.

**Reply by barbudor** [reply]
When you install Tasmota on the ESP of the Dimmer, an alternative firmware from [James Sturton](https://github.com/jamesturton/shelly-dimmer-stm32) is installed on the STM32 that controls the dimming.
You can read the following [thread](https://github.com/jamesturton/shelly-dimmer-stm32/issues/7) about problems with no-neutral.
You can also read and join this discussion with [CReiner](https://github.com/arendst/Tasmota/discussions/11050) whose's goal is to make Tasmota to work with Shelly's original STM32 firmware.

But so far in my case, I was unable to get a proper behavior of the dimmer without neutral even with full Shelly's original ESP+STM firmwares.
